process/smartContract/builtInFunctions/factory.go

```diff
@@ -0,0 +1,89 @@
+package builtInFunctions
+
+import (
+	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/ElrondNetwork/elrond-go/core/check"
+	"github.com/ElrondNetwork/elrond-go/process"
+	"github.com/mitchellh/mapstructure"
+)
+
+// claimDeveloperRewardsFunctionName is a constant which defines the name for the claim developer rewards function
+const claimDeveloperRewardsFunctionName = "ClaimDeveloperRewards"
```

Comment:
I think these should be exported in order to be useful

Reply:
there is a PR in work for this - to come from config.